packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json

```diff
@@ -641,6 +641,26 @@
                     }
                 }
             },
+            "BesuPrivateTransactionConfig" : {
```

Comment:
@travis-payne Please open a follow-up issue to have the code support the other possible parameters as well (privacyGroupId and privateKey)

See the index.d.ts file for reference, quoting from there:

```typescript
 export interface ISendRawTransactionOptions extends ISendGenericTransactionOptions, TransactionConfig {
        /**
         * Private Key used to sign transaction with
         */
        readonly privateKey: string;
        /**
         * Enclave public key
         */
        readonly privateFrom: string;
        /**
         * Enclave keys to send the transaction to
         */
        readonly privateFor: string[];
        /**
         * Enclave id representing the receivers of the transaction
         */
        readonly privacyGroupId?: string;
        /**
         * (Optional) : If not provided, will be calculated using`eea_getTransctionCount`
         */
        readonly nonce?: number;
        /**
         * The address to send the transaction
         */
        readonly to?: string;
        /**
         * Data to be sent in the transaction
         */
        readonly data: string;
    }
```

Reply:
I think private key isn't needed, users add that as part of their Web3SigningCredential, but privacyGroupId is defo needed.